gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -230,11 +230,14 @@ MM_VerboseHandlerOutput::handleInitializedRegion(J9HookInterface** hook, uintptr
 	MM_InitializedEvent* event = (MM_InitializedEvent*)eventData;
 	MM_VerboseWriterChain* writer = _manager->getWriterChain();
 	MM_EnvironmentBase* env = MM_EnvironmentBase::getEnvironment(event->currentThread);
+	const char *arrayletDoubleMappingStatus = event->isArrayletDoubleMappingEnabled ? "true" : "false";
+	const char *arrayletDoubleMappingRequested = event->isArrayletDoubleMappingRequested ? "true" : "false";
 
 	writer->formatAndOutput(env, 1, "<region>");
 	writer->formatAndOutput(env, 2, "<attribute name=\"regionSize\" value=\"%zu\" />", event->regionSize);
 	writer->formatAndOutput(env, 2, "<attribute name=\"regionCount\" value=\"%zu\" />", event->regionCount);
 	writer->formatAndOutput(env, 2, "<attribute name=\"arrayletLeafSize\" value=\"%zu\" />", event->arrayletLeafSize);
+	writer->formatAndOutput(env, 2, "<attribute name=\"isArrayletDoubleMappingEnabled\" status value=\"%s\" | requested value=\"%s\"/>", arrayletDoubleMappingStatus, arrayletDoubleMappingRequested);
```

Comment:
- ifdef for supported platforms only
- report requested first, status second
- isArrayletDoubleMappingEnabled simplify to just arrayletDoubleMapping
- simplify by just stating 'requested=' and 'status=", no need for 'value' word (not expert on XML but attribute name is only one word typically, if not always)
- status attriblue value should be 'enabled/disabled'. or attributte name should be 'enabled', and values would stay 'true/false'. whichever of the two approaches you prefer
- no need for '|' separator